build.gradle

```diff
@@ -30,39 +31,59 @@ repositories {
 
 
 apply plugin: 'org.junit.platform.gradle.plugin'
-//apply plugin: 'kotlin'
+apply plugin: 'idea'
 
 group = 'com.buildit'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = 1.8
 
+sourceSets {
+    'test-e2e' {
+        kotlin {
+            srcDir files('src/test-e2e/kotlin')
+        }
+        resources.srcDir files('src/test-e2e/resources')
+    }
+}
+
+idea {
+    module {
+        testSourceDirs += files('src/test-e2e/kotlin')
+    }
+}
+
 compileKotlin {
-	kotlinOptions.jvmTarget = "1.8"
+    kotlinOptions.jvmTarget = "1.8"
 }
+
 compileTestKotlin {
-	kotlinOptions.jvmTarget = "1.8"
+    kotlinOptions.jvmTarget = "1.8"
 }
 
 repositories {
-	mavenCentral()
+    mavenCentral()
+}
+
+springBoot {
+    buildInfo()
```

Comment:
kind of nice to include build information in the jar file for traceability
https://docs.spring.io/spring-boot/docs/current/reference/html/howto-build.html